test/unittest/backends/mongodb/test_queries.py

```diff
@@ -96,100 +96,130 @@ def test_insert_from_uris(db, image_url):
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_like(db):
-    collection = Collection('documents')
+    collection = Collection("documents")
     r = db.execute(collection.find_one())
     query = collection.like(
-        r=Document({'x': r['x']}),
-        vector_index='test_vector_search',
+        r=Document({"x": r["x"]}),
+        vector_index="test_vector_search",
     ).find()
     s = next(db.execute(query))
-    assert r['_id'] == s['_id']
+    assert r["_id"] == s["_id"]
 
 
-@pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_insert_one(db):
     # MARK: empty Collection + a_single_insert
-    collection = Collection('documents')
-    a_single_insert = get_new_data(db.encoders['torch.float32[32]'], 1, update=False)[0]
+    collection = Collection("documents")
+    a_single_insert = get_new_data(db.encoders["torch.float32[32]"], 1, update=False)[0]
     out, _ = db.execute(collection.insert_one(a_single_insert))
-    r = db.execute(collection.find({'_id': out[0]}))
+    r = db.execute(collection.find({"_id": out[0]}))
     docs = list(r)
-    assert docs[0]['x'].x.tolist() == a_single_insert['x'].x.tolist()
+    assert docs[0]["x"].x.tolist() == a_single_insert["x"].x.tolist()
 
 
-@pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_delete_one(db):
     # MARK: random data (change)
-    collection = Collection('documents')
+    collection = Collection("documents")
     r = db.execute(collection.find_one())
-    db.execute(collection.delete_one({'_id': r['_id']}))
+    db.execute(collection.delete_one({"_id": r["_id"]}))
     with pytest.raises(StopIteration):
-        next(db.execute(Collection('documents').find({'_id': r['_id']})))
+        next(db.execute(Collection("documents").find({"_id": r["_id"]})))
 
 
-@pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_find(db):
-    collection = Collection('documents')
+    collection = Collection("documents")
     r = db.execute(collection.find().limit(1))
     assert len(list(r)) == 1
     r = db.execute(collection.find().limit(5))
     assert len(list(r)) == 5
 
 
-@pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_find_one(db):
-    r = db.execute(Collection('documents').find_one())
+    r = db.execute(Collection("documents").find_one())
     assert isinstance(r, Document)
 
 
-@pytest.mark.skipif(not torch, reason='Torch not installed')
+@pytest.mark.skipif(not torch, reason="Torch not installed")
 def test_aggregate(db):
-    r = db.execute(Collection('documents').aggregate([{'$sample': {'size': 2}}]))
+    r = db.execute(Collection("documents").aggregate([{"$sample": {"size": 2}}]))
```

Comment:
Does this come from a linter or a text-editor?

Reply:
Most probably from the vs-codes ruff. Gone back to `'`